docs/source/blogging.rst

```diff
@@ -19,23 +21,27 @@ Below are the steps you need to perform in order to get started.
 Request an invite link
 ----------------------
 
-To get an invite link, ping `Rahul Jha [:RJ722] <https://rj722.github.io>`_ on Slack,
-requesting for an invite link.
+To get an invite link, ping `Rahul Jha [:RJ722] <https://rj722.github.io>`_ on
+Slack, asking for an invite link.
 
-After you've received the link, open it to register yourself on https://blogs.amuoss.tk
-with your email (so that you don't get locked out when you forget your password).
+After you've received the link, open it to register yourself on
+https://blogs.amuoss.tk with your email (so that you don't get locked out when
+you forget your password).
 
 Configure your blog space
 -------------------------
 
 Make your blog visible to everyone
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-By default, all the posts you write are stored as drafts in your blog.
-To make them public, follow these steps -
- - navigate to `your blog <https://blogs.amuoss.tk/me/c/>`_
- - open the `customize <https://blogs.amuoss.tk/me/c/roshan>`_ link. This opens up your blog settings page.
- - Under the *Publicity* section, change *Private* to **Public**.
+By default, all the posts you write are unlisted (i.e. they do not appear on
+the https://blogs.amuoss.tk/read).
```

Comment:
```suggestion
the `reader <https://blogs.amuoss.tk/read>`).
```

Reply:
cc @RJ722

Reply:
This was intentional because because when we mention it for the first time, they won't know what a reader is.

Reply:
I need to remove the "the" though.